Makefile

```diff
@@ -331,6 +331,9 @@ define deploy-local-cluster
 	@echo "\n$(YELLOW)Deploy test apps $(NC)"
 	$(call deploy-test-apps)
 
+	@echo "\n$(YELLOW)Expose k8gb for DNS traffic $(NC)"
+	$(call expose-k8gb-dns)
```

Comment:
Insn't it conflicting with https://github.com/AbsaOSS/k8gb/blob/master/deploy/ingress/nginx-ingress-values.yaml#L38 ?

Reply:
@k0da indeed, nice spot! Amended.